src/test/scala/dropInconsistentAssignments.scala

```diff
@@ -63,11 +64,10 @@ case class inconsistentAssignmentsFilter(
   }
 
   /* This threshold determines minimum percentage of the taxon's parent's count compared to the total count */
-  // FIXME: this constant needs a review
-  val countsPercentageMinimum: Double = 75.0 // % minimum
+  val countsPercentageMinimum: Double = 0.75 // 75% minimum
 
   /* This predicate determines whether to drop the taxon or to keep it */
-  def predicate(countsMap: Map[Taxon, (Int, Seq[Taxon])], totalCount: Int): Taxon => Boolean = { taxon =>
+  def predicate(countsMap: Map[Taxon, (Int, Lineage)], totalCount: Int): Taxon => Boolean = { taxon =>
 
     countsMap.get(taxon)
       /* First we find `taxon`'s ancestor ID (2 levels up) */
```

Comment:
better put this and the minimum ratio as values at the beginning (or better, make a method with those two values as input). These values are likely to change / be adjusted in future releases.